README.md

```diff
@@ -9,22 +9,40 @@ A hello world for Google Cloudbuild.
 Running the [trigger image](https://hub.docker.com/r/r6by/trigger-gcp-cloudbuild/) asks Google cloudbuild to create and store a second image defined by the included `source` directory. Running the built image should output:
 > Built by GCP Cloudbuild
 
-## Test setup
+## IAM setup
 
-- Create GCP service account with "Cloud Container Builder Editor" and "Storage Object Admin" roles (check the box "Furnish a new private key" and select key type "JSON".
-- Note the `[/PATH/TO/SERVICE/ACCOUNT/KEYFILE].json`
+1. Create a GCP service account:
+
+    ```console
+    $ gcloud iam service-accounts create trigger-gcb
+    ```
+
+1. Add the "Cloud Container Builder Editor" and "Storage Object Admin" roles to the service account.
+
+    ```console
+    $ export SA_EMAIL=$(gcloud iam service-accounts list --filter="name:trigger-gcb" --format='value(email)')
+    $ export PROJECT=$(gcloud info --format='value(config.project)')
+    $ gcloud projects add-iam-policy-binding $PROJECT --role roles/cloudbuild.builds.editor --member serviceAccount:$SA_EMAIL ---role roles/storage.admin
```

Comment:
Need to remove extra hyphen from `---role`. Also, is it a good idea to add `--display-name` too? Or does that matter. I'm pretty new to Google's IAM